src/Features/CSharp/Portable/ReplaceDocCommentTextWithTag/CSharpReplaceDocCommentTextWithTagCodeRefactoringProvider.cs

```diff
@@ -11,15 +12,37 @@ namespace Microsoft.CodeAnalysis.CSharp.ReplaceDocCommentTextWithTag
     internal class CSharpReplaceDocCommentTextWithTagCodeRefactoringProvider :
         AbstractReplaceDocCommentTextWithTagCodeRefactoringProvider
     {
+        private static HashSet<string> s_keywords = new HashSet<string>
+        {
+            SyntaxFacts.GetText(SyntaxKind.NullKeyword),
+            SyntaxFacts.GetText(SyntaxKind.StaticKeyword),
+            SyntaxFacts.GetText(SyntaxKind.VirtualKeyword),
+            SyntaxFacts.GetText(SyntaxKind.TrueKeyword),
+            SyntaxFacts.GetText(SyntaxKind.FalseKeyword),
+            SyntaxFacts.GetText(SyntaxKind.AbstractKeyword),
+            SyntaxFacts.GetText(SyntaxKind.SealedKeyword),
+            SyntaxFacts.GetText(SyntaxKind.AsyncKeyword),
+            SyntaxFacts.GetText(SyntaxKind.AwaitKeyword)
+        };
+
         protected override bool IsXmlTextToken(SyntaxToken token)
             => token.Kind() == SyntaxKind.XmlTextLiteralToken ||
                token.Kind() == SyntaxKind.XmlTextLiteralNewLineToken;
 
-        protected override bool IsAnyKeyword(string text)
-            => SyntaxFacts.GetKeywordKind(text) != SyntaxKind.None ||
-               SyntaxFacts.GetContextualKeywordKind(text) != SyntaxKind.None;
+        protected override bool IsInXMLAttribute(SyntaxToken token)
+        {
+            return (token.Parent.Kind() == SyntaxKind.XmlCrefAttribute
```

Comment:
this seems very specific list. would be better if it is done reversed way like listing kinds that should work? if statements above, isXmlText and etc seems explicitly checking allowed case.

Reply:
actually I might miss read, this might be only cases for token to be in xml attributes

Reply:
Correct, this is a list of the keywords that we .want to trigger on.